databricks/koalas/frame.py

```diff
@@ -1764,14 +1764,10 @@ def to_latex(
             kdf._to_internal_pandas(), self.to_latex, pd.DataFrame.to_latex, args
         )
 
-    def to_markdown(self, buf=None, mode=None, max_rows=None):
+    def to_markdown(self, buf=None, mode=None):
         """
         Print DataFrame in Markdown-friendly format.
 
-        .. note:: This method should only be used if the resulting Pandas object is expected
-                  to be small, as all the data is loaded into the driver's memory. If the input
-                  is large, set max_rows parameter.
```

Comment:
We should notice users that this API loads the all data into the driver's memory.

So, you can remove the only `max_rows` part:

```python
        .. note:: This method should only be used if the resulting Pandas object is expected
                  to be small, as all the data is loaded into the driver's memory.
```

Reply:
okay. I will recover that part.